src/test/java/com/amazon/opendistroforelasticsearch/sql/intgtest/NestedFieldQueryTest.java

```diff
@@ -341,6 +355,40 @@ public void groupByNestedAndRegularField() {
         return kv("myNum", is(value));
     }
 
+    private Matcher<SearchHit> myNum(int[] values) {
+
+        return new BaseMatcher<SearchHit>() {
+
+            @Override
+            public boolean matches(Object item) {
+
+                if (item instanceof SearchHit) {
+                    final SearchHit hit = (SearchHit)item;
+                    List<Integer> actualValues = (ArrayList<Integer>)hit.getSourceAsMap().get("myNum");
+
+                    if (actualValues.size() != values.length) {
+                        return false;
+                    }
+
```

Comment:
LGTM. np: I see a lot of extra empty lines in this function. See if it can be removed without disturbing the overall structure of the class.

Reply:
Removed some new lines.